src/components/messageSpace/messageSpace.tsx

```diff
@@ -9,9 +9,13 @@ import Icon from '../icon'
 import MessageCanvas, {
   type MessageContainerProps,
 } from '../messageCanvas/messageCanvas'
-import type { ComponentMap, ThreadableMessage } from '../types'
+import type { ComponentMap, ThreadableMessage, WebSocketClient } from '../types'
 
 export interface MessageSpaceProps extends MessageContainerProps {
+  /** WebSocket connection to send and receive messages to and from a backend. Some components in this library require this value in order to be functional, such as the `Question` component. */
```

Comment:
```suggestion
  /** WebSocket connection to send and receive messages to and from a backend. This can be useful for component interactions, for example, to send filter conditions, or user location, etc */
```